packages/compiler/core/projector.ts

```diff
@@ -641,17 +633,4 @@ export function createProjector(
     projectedTypes.set(type, clone);
     return clone;
   }
-
-  /**
-   * Project the given type by projecting the parent type first.
-   * @param type Type to project.
-   * @param parentType Parent type that should be projected first.
-   * @returns Projected type
-   */
-  function projectViaParent(type: Type, parentType: Type): Type {
```

Comment:
Not needed anymore?  Seems like I remember this being added to fix a projection bug we found in Azure.Core, but I could be wrong.

Reply:
Yeah, that fixed it at the time but this ended up building on the problem. I do remember adding a regression test to make sure we wouldn't break it again. I'll double check this PR works fine in teh cadl-azure repo before merging

Reply:
Ok, found the problem and  it is working now